WalletWasabi.Fluent/ViewModels/Wallets/Send/SendViewModel.cs

```diff
@@ -408,6 +438,7 @@ protected override void OnNavigatedFrom(bool isInHistory)
 			base.OnNavigatedFrom(isInHistory);
 			_lastXAxisCurrentValue = XAxisCurrentValue;
 			_transactionInfo.ConfirmationTimeSpan = CalculateConfirmationTime(_lastXAxisCurrentValue);
+			_ = _qrReader.StopScanningAsync();
```

Comment:
It is just a warning elimination, but in fact, it is the same as a non awaited Task.Run, do not do this.
Use `RxApp.MainThreadScheduler.Schedule(async () => await _qrReader.StopScanningAsync());` instead, RxApp is awaited somewhere, so if something happen on that thread we will know about it.